site/index.html

```diff
@@ -1,14 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Your Website Name</title>
-    <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
-    <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
-    <link rel="stylesheet" href="styles.css" type="text/css" />
-  </head>
-  <body>
-    <nav><a href="/">Home</a><a href="/about">About</a></nav>
-    <h1>Homepage</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Welcome to Boston Arts Academy</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <div class="header-box">
+    <h1>Artist-scholars designing their own future.</h1>
+    <div>view their wokr</div>
+  </div>
+  <div class="upcoming events">
+    <h2>Upcoming events</h2>
+    <div>view all</div>
+    <div>
+      <div>
+        <div>March 10</div>
+        <div>Applications Are Due: Hoope to see you nect year!</div>
+      </div>
+      <div>
+        <div>March 17</div>
+        <div>The Art of Connection: A Virtual Symposium</div>
+      </div>
+      <div>
+        <div>June 23</div>
+        <div>Aretha Franklin Tribute: Gospel Years</div>
+      </div>
+      <div class="news">
+        <div>News</div>
+        <div>view All</div>
+      </div>
+      <div class="box 2" <h3>174 Ipswich Street</h3>
+        <div>Tour the New BAA</div>
+        <div> Professor Jonah Harding Wins Genius Grant</div>
+        <div>Alum Franny Hall on nine Years Leading a Non-Profit</div>
+        <h4>Student Spotlight</h4>
```

Comment:
Try making all similar headers be the same header level. For example, you have Upcoming Events on line 19 is a h2. When I look at the UX PDF it looks like Student Spotlight is the same level of header as "Upcoming Events". When headers look like they are the same level you can make them the same level header tag.